native/src/components/PoiFiltersModal.tsx

```diff
@@ -127,11 +127,7 @@ const PoiFiltersModal = ({
           </TileRow>
         </Section>
         <Section>
-          {poisCount !== 0 ? (
-            <StyledTextButton onPress={closeModal} text={t('showPois', { count: poisCount })} />
-          ) : (
-            <StyledTextButton onPress={closeModal} text={t('noPois')} disabled />
-          )}
+          <StyledTextButton onPress={closeModal} text={t('showPois', { count: poisCount })} />
```

Comment:
Nice, now we are nearly there. The only thing missing is passing the `disabled` prop and deciding correctly when it should be disabled :) Great work so far :tada: